spec/features/state_file/complete_intake_spec.rb

```diff
@@ -124,6 +125,10 @@
       fill_in "state_file_az_subtractions_form_armed_forces_wages_amount", with: "100"
       click_on I18n.t("general.continue")
 
+      expect(page).to have_text I18n.t("state_file.questions.federal_extension_payments.edit.title")
+      choose I18n.t("general.affirmative")
+      click_on I18n.t("general.continue")
+
```

Comment:
should i test this page in the complete intake spec now? or just wait for the flipper to be completely live & then these tests should start failing...

Reply:
talked it out during stand up -- best to test out things that aren't yet on production in test